revdep/problems.md

```diff
@@ -0,0 +1,17881 @@
+
+* Version: 1.0.1
+* GitHub: NA
+* Source code: https://github.com/cran/DImodelsVis
+* Date/Publication: 2024-02-26 14:10:15 UTC
+* Number of recursive dependencies: 113
+
+Run `revdepcheck::cloud_details(, "DImodelsVis")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking examples ... ERROR
+    ```
+    Running examples in ‘DImodelsVis-Ex.R’ failed
+    The error most likely occurred in:
+    
+    > ### Name: conditional_ternary
+    > ### Title: Conditional ternary diagrams
+    > ### Aliases: conditional_ternary
+    > 
+    > ### ** Examples
+    > 
+    > library(DImodels)
+    ...
+     11.       │ └─base::withCallingHandlers(...)
+     12.       └─ggplot2 (local) f(l = layers[[i]], d = data[[i]])
+     13.         └─l$compute_position(d, layout)
+     14.           └─ggplot2 (local) compute_position(..., self = self)
+     15.             └─self$position$use_defaults(data, self$aes_params)
+     16.               └─ggplot2 (local) use_defaults(..., self = self)
+     17.                 └─ggplot2:::check_aesthetics(new, nrow(data))
+     18.                   └─cli::cli_abort(...)
+     19.                     └─rlang::abort(...)
+    Execution halted
+    ```
+
+*   checking re-building of vignette outputs ... ERROR
+    ```
+    Error(s) in re-building vignettes:
+    --- re-building ‘DImodelsVis-with-complex-models.Rmd’ using rmarkdown
+    ```
+
+# directlabels
+
+<details>
+
+* Version: 2024.1.21
+* GitHub: https://github.com/tdhock/directlabels
+* Source code: https://github.com/cran/directlabels
+* Date/Publication: 2024-01-24 19:20:07 UTC
+* Number of recursive dependencies: 80
+
+Run `revdepcheck::cloud_details(, "directlabels")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking re-building of vignette outputs ... ERROR
+    ```
+    Error(s) in re-building vignettes:
+    --- re-building ‘examples.Rmd’ using knitr
+    ```
+
+# distributions3
+
+<details>
+
+* Version: 0.2.2
+* GitHub: https://github.com/alexpghayes/distributions3
+* Source code: https://github.com/cran/distributions3
+* Date/Publication: 2024-09-16 16:20:02 UTC
+* Number of recursive dependencies: 89
+
+Run `revdepcheck::cloud_details(, "distributions3")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking tests ... ERROR
+    ```
+      Running ‘testthat.R’
+    Running the tests in ‘tests/testthat.R’ failed.
+    Complete output:
+      > library(testthat)
+      > library(distributions3)
+      
+      Attaching package: 'distributions3'
+      
+      The following object is masked from 'package:stats':
+      
+    ...
+      
+      ══ Failed tests ════════════════════════════════════════════════════════════════
+      ── Failure ('test-plot.R:50:3'): ggplot2 implementation works ──────────────────
+      `print(gg1)` produced warnings.
+      ── Failure ('test-plot.R:56:3'): ggplot2 implementation works ──────────────────
+      `print(gg3)` produced warnings.
+      
+      [ FAIL 2 | WARN 14 | SKIP 0 | PASS 2766 ]
```

Comment:
Expecting functions to be silent will break upon lifecycle changes. These should probably be snapshot tests on their end.